apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -111,94 +122,162 @@ static int write_file(int fd, size_t bufsize)
 
 	ret = write(fd, (void *)&g_buf, bufsize);
 	if (ret != bufsize) {
-		printf("write_file error ret : %d errno : %d\n", ret, errno);
+		printf("write_random_data_file error ret : %d errno : %d\n", ret, errno);
 		return ERROR;
 	}
 
-	return ret;
+	return OK;
 }
 
 /* small file uses buffer size same as stat.st_blksize */
-int create_file(char *path, ssize_t bufsize)
+static int create_file(char *path, ssize_t bufsize, file_type_t type)
 {
 	int fd;
-	int len;
-	int total_len = 0;
+	int ret;
 	struct stat st;
+
+	/* If file is already exist and written fully, just return OK */
 	if (stat(path, &st) == OK) {
-		printf("path %s already exist size : %d\n", path, st.st_size);
+		printf("File already exist path : %s size : %d\n", path, st.st_size);
 		return OK;
+
 	}
-			
-	fd = open(path, O_CREAT | O_EXCL | O_WROK);
+	
+	fd = open(path, O_CREAT | O_WROK);
```

Comment:
Exclusivity while writing to src file not needed?

Reply:
see above line, it checked by stat.